src/pkg/data/consumers/governor/exporter_test.go

```diff
@@ -51,11 +53,11 @@ func TestSendReportToGovernorSuccess(t *testing.T) {
 						ImageID: imageID,
 					}}}}}}}}}}},
 		ClusterID: clusterID,
-		ApiToken:  apiToken,
 		ApiClient: mockClient,
```

Comment:
Add UT for -ve cases of SendReportGovernor w.r.t cspProvider and GovernorAccessToken

Reply:
Done.